src/core.js

```diff
@@ -755,11 +755,30 @@ define([
             }
 
             if (window.analytics) {
-                window.analytics.usage(
-                    "Question Reference",
-                    "Drag and Drop",
-                    _this.data.core.currentlyEditedProperty
-                );
+                var targetType;
+                switch (target[0].id) {
+                    case 'property-relevantAttr':
+                        targetType = "Display";
+                        break;
+                    case 'property-constraintAttr':
+                        targetType = "Validation";
+                        break;
+                    case 'property-calculateAttr':
+                        targetType = "Calculation";
+                        break;
+                    default:
+                        targetType = "Expression Editor";
+                        break;
+                }
+                if (path.contains('casedb')) {
```

Comment:
I wouldn't depend on it since it's not a precondition of this function that `mug` is defined for all dropped paths that are form properties.

For now I'd recommend implementing a function on the form that you can call like `_this.data.core.form.isFormReference(path)` and swap the branches in this `if`-block. In that function you can
```js
isFormReference: function (path) {
    return this.getMugByPath(path) ? true : false;
},
```
That way you can call the same function below in javaRosa.js and we can easily update it later if the criteria changes.

@emord wrote:
> This is purely for the databrowser drag/drop...

Isn't this function also called on drag/drop from the question tree?